services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -464,6 +450,66 @@ private UserConsentModel createConsent(ClientModel client, ConsentRepresentation
         return consent;
     }
 
+    @Path("/applications")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @NoCache
+    public Response applications() {
```

Comment:
Is there a query param to control the locale? Should probably have an optional ui_locales like authorization code flow requests has.